test/tool-option-parsing.py

```diff
@@ -223,9 +225,14 @@ def test_compile_keymap_args(self):
                 self.xkbcli_compile_keymap.run_command_success(args)
 
     def test_compile_keymap_rmlvo(self):
-        for rmlvo in rmlvos:
+        def run(rmlvo):
             with self.subTest(rmlvo=rmlvo):
-                self.xkbcli_compile_keymap.run_command_success(rmlvo)
+                return self.xkbcli_compile_keymap.run_command_success(rmlvo)
+
+        with concurrent.futures.ThreadPoolExecutor() as executor:
```

Comment:
By default the number of threads is `os.cpu_count() * 5` ([ref](https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.ThreadPoolExecutor)). Just guessing, but I think just `max_workers=os.cpu_count()` would be more suitable in this case.

Reply:
The default is `min(32, os.cpu_count() + 4)` (it’s an addition, not a multiplication), so in the range `[4, 32]`.